src/app/handlers/new_member.handler.ts

```diff
@@ -0,0 +1,33 @@
+import { GuildMember } from 'discord.js';
+import { IContainer, IHandler } from '../../common/types';
+
+export class NewMemberHandler implements IHandler {
+  private _unverifiedRole: any = undefined;
+  private _unacknowledgedRole: any = undefined;
```

Comment:
we don't *have* to use `any` here. it's better to use something like

`RoleType | undefined` (not sure what discord role type is).

a common abstraction that people do, and might be worth to transition into now, is creating a type `Maybe`

```
type Maybe<T> = T | undefined | null;
``` 

and then ur type for this variable would just be `Maybe<RoleType>`

Reply:
Put this in the same file on top of the class? or?

Reply:
types.ts makes sense